_posts/en/podcast/2023-06-15-newsletter-recap.md

```diff
@@ -0,0 +1,1345 @@
+silent payments is in getting feedback from the community?
+
+**Josie Baker**: Yeah, I think the biggest misconception, and sometimes it's a
+misconception and also a criticism, is that this doesn't work for mobile
+clients.  That was feedback that was brought up early on, I think March 2022,
+some people raised that point.  And basically what that boils down to is,
+Ruben's mentioned this as a very old idea of these reusable payment codes, and I
+would group all the proposals that have existed as either schemes that use
+notifications, onchain notifications, and schemes that don't.  And silent
+payments, as far as I'm aware, is one of the first ones that doesn't use
+notifications.  And the way we get away with not using notifications is we have
+to scan the chain.
+
+So for a full node, this is not a problem.  The job of a full node is to look at
+every new, incoming transaction in every block and decide whether or not it's
+valid and whether or not to add it to its local longest chain.  For a mobile
+client, you don't have access to every transaction in every block and you don't
+want access to every transaction in every block.  So the criticism with sound
+payments was, well if you're switching to something that requires scanning, it's
+not going to work for mobile clients.
+
+I'm pretty confident that it will work for mobile clients, but with the
+trade-off that it's going to require more bandwidth.  So, I think we wrote a
+pretty nice section on the BIP to address this of like, "Mobile client support
+is going to be tricky, but we definitely think it's possible".  And I think the
+clearest path forward is looking how we can do this with BIP158.  BIP158, I
+don't think it's seen a lot of adoption in wallets right now, so that's kind of
+an open question too, like how well does BIP158 work with mobile clients?  But
+there's quite a few other ideas that we're kicking around.
+
+I think there's definitely a path towards getting something that works with the
+Electrum protocol, but ultimately it's going to come down to a bandwidth
+trade-off.  You're going to have something that works with your mobile wallet in
+a very privacy-preserving way, but with the cost of using more bandwidth.  And I
+think a lot of the mobile wallets today are either backed by their own full
+node, which then, in my mind, it no longer classifies as a mobile wallet, or
+they're making some sort of trusted trade-off with some other counterparty to
+give them that data.
+
+So with silent payments, I'd say it's not much different.  We have to look at,
+are we going to trust someone to do the scanning for us, whether it be my own
+node; or at the expense of more bandwidth, can I actually run a mobile client in
+a privacy preserving way?  I think that's the biggest one.  I don't know, Ruben,
+if you have any other ones you want to add or any color to add to that?
+
+<!-- skip-duplicate-words-test -->**Ruben Somsen**: Yeah, no, I agree that that's the one that a lot of people
```

Comment:
Do you think we should exclude the transcripts from the duplicate words test?  Or exclude "that that" from the test?

Reply:
@harding I think that might be best